lib/gears/math.lua

```diff
@@ -79,5 +79,16 @@ function gmath.round(x)
     return math.floor(x + 0.5)
 end
 
+--- Round a number to a 10s place
+-- @class function
+-- @name roundf
+-- @tparam number v
+-- @tparam[opt=1] number bracket 10s place to round to
+function gmath.roundf(v, bracket)
+	-- From lua-users.org SimpleRound
+	local function sign(v) return (v >= 0 and 1) or -1 end
```

Comment:
The `sign` function is used just once and thus does not need to be a function.

Also, unit tests would be nice (also for the other functions in this PR)